### PR TITLE
Taxonomy Browser UI improvements

### PR DESCRIPTION
**Pull request by madawas**

### CODE REVIEW

File: apps/publisher/themes/default/js/taxonomy-browser.js [hunk 1]
```diff
@@ -164,7 +165,9 @@ var loadTaxonomyChildren = function (taxonomyName, element, dataWindow) {
  */
 var initTaxonomyBrowser = function (appliedTaxonomy) {
     $(TAXONOMY_BROWSER).hide();
-    $('#taxonomy-list')[0].value = '';
+    if($('#taxonomy-list')[0]) {
```
Comment (cnapagoda) [inline apps/publisher/themes/default/js/taxonomy-browser.js]: Formatting issue

Comment (madawas) [inline apps/publisher/themes/default/js/taxonomy-browser.js]: Fixed in 7d7f0f5285b54aec441b2443fea7f4acb5b233de

File: apps/publisher/themes/default/js/taxonomy-browser.js [hunk 2]
```diff
@@ -282,34 +285,58 @@ $(function () {
...
             $(SELECTED_CONTAINER).show();
         }
+        $(this).closest('li').append(' <button class="btn btn-remove">'
+            + '<span class="icon fw-stack"><i class="fw fw-minus fw-stack-1x"></i>'
+            + '<i class="fw fw-ring fw-stack-2x"></i></span>  Remove</button>');
+        $(this).remove();
+
+    });
+
+    $(TAXONOMY_SELECT).on('click', COLUMN_SELECTOR + ' li.leaf > button.btn-remove', function (e) {
```
Comment (lahirue) [inline apps/publisher/themes/default/js/taxonomy-browser.js]: Its better to have method description

Comment (madawas) [inline apps/publisher/themes/default/js/taxonomy-browser.js]: @lahirue I have already added an description in 47a37a0dd9e9a41be0dcced7f68650998e776226

File: apps/publisher/themes/default/js/taxonomy-browser.js [hunk 3]
```diff
@@ -282,34 +285,58 @@ $(function () {
         $(BREADCRUMB_SELECTOR).html($(BREADCRUMB_SELECTOR + ' > li').first());
         $(COLUMN_SELECTOR + ' ul > li').each(function () {
             if ($(this).hasClass('active')) {
-                displayValue.push($('a', this).text());
+                // displayValue.push($('a', this).text());
```
Comment (thushara35) [inline apps/publisher/themes/default/js/taxonomy-browser.js]: Better if you can explain by adding a comment.

Comment (madawas) [inline apps/publisher/themes/default/js/taxonomy-browser.js]: @thushara35 Added a comment and explained in #841 (9918ae8)

**Pull request merged**